Extension/ASPNET-MVC-Extension/Overview.md

```diff
@@ -23,15 +23,15 @@ The Syncfusion provides the following extension supports in Visual Studio:
 
 **No project selected in Visual Studio**
 
-![Syncfusion Menu when No project selected in Visual Studio](Overview-images/Syncfusion_Menu_OverView1.png)
+![Syncfusion Menu when No project selected in Visual Studio for aspnet.mvc extension overview](overview-images/aspnet-mvc-extension-syncfusion-menu-overview.png)
```

Comment:
reduce length of the alt text and image name

Reply:
Resolved the comment